pages/common/gh-release.md

```diff
@@ -0,0 +1,20 @@
+# gh-release
+
+> Manage GitHub pull requests from the command line.
+> More information: <https://cli.github.com/manual/gh_release_delete>.
+
+- List GitHub Releases in a repository (Default limit 30):
```

Comment:
Maybe we can say `GitHub repository` in the first example, but I think repeat `GitHub` in every example is noisy
```suggestion
- List releases in a GitHub repository, limit 30 items:
```

Reply:
Otherwise:
```suggestion
- List the last 30 releases in a GitHub repository:
```